### PR TITLE
Add Codes and Recourse items

### PR DESCRIPTION
**Pull request by yash244466666**
Add Codes in the following files.

[src](https://github.com/yash244466666/To-Do-List/tree/main/src)
- [index.html](https://github.com/yash244466666/To-Do-List/blob/main/src/index.html)
- [index.js](https://github.com/yash244466666/To-Do-List/blob/main/src/index.js)
- [style.css](https://github.com/yash244466666/To-Do-List/blob/main/src/style.css)

Code Output
[dist](https://github.com/yash244466666/To-Do-List/tree/main/dist)

- [bundle.js](https://github.com/yash244466666/To-Do-List/blob/main/dist/bundle.js)
- [index.html](https://github.com/yash244466666/To-Do-List/blob/main/dist/index.html)

### CODE REVIEW

**Review by Davidosky007 — COMMENTED** [review]
Hi @yash244466666 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesnt have a descriptive readme file, Kindly Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by yash244466666** [comment]
Add and edit the following files for the updated readme.

- [README.md](https://github.com/yash244466666/To-Do-List/blob/addCodesForDayTow/README.md)
- [MIT.md](https://github.com/yash244466666/To-Do-List/blob/addCodesForDayTow/MIT.md)
- [murple_logo.png](https://github.com/yash244466666/To-Do-List/blob/addCodesForDayTow/murple_logo.png)

File: README.md [hunk 1]
```diff
@@ -1,4 +1,214 @@
...
+> Mention all of the collaborators of this project. <br>
+
+
+👤 **Author 1**
+- Yash
+- GitHub: [Yash](https://github.com/yash244466666)
+- LinkedIn: [Yash Solo](https://www.linkedin.com/in/yash-solo)
+- Twitter: [Yash Solo](https://twitter.com/yash_solo000)
 
 # Project Name
```
Comment (TedLivist) [inline README.md]: - [ ] Your README combines the old and newer readme. Kindly use only the newer version as shared with you earlier.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,214 @@
...
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
```
Comment (TedLivist) [inline README.md]: - [ ] Kindly make your README more professional. Kindly include:
  - Prerequisites users need to have to set up the project
  - How to set up and run the project locally.

File: src/index.js [hunk 3]
```diff
@@ -1,13 +1,45 @@
...
+        <br>
+      </div>
 
-  return element;
-}
+    </div>
+    `;
+  });
+  document.getElementById('task').innerHTML = result;
+};
```
Comment (TedLivist) [inline src/index.js]: - [ ] Kindly ensure your layout matches the provided one.

  Your Implementation | Required layout
  :---:|:---:
  eieo | kdkd

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @yash244466666,

Good job so far!
✔️ App is properly set up with webpack
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yash244466666** [comment]
Add and update the following files as instructed

- [x] [README.md](https://github.com/yash244466666/To-Do-List/blob/addCodesForDayTow/README.md)


[src](https://github.com/yash244466666/To-Do-List/tree/addCodesForDayTow/src)
- [x] [style.css](https://github.com/yash244466666/To-Do-List/blob/addCodesForDayTow/src/style.css)

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @yash244466666,

✔️ Nice layout

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**